admin/load.php

```diff
@@ -636,3 +647,44 @@ function perflab_plugin_admin_notices() {
 	}
 }
 add_action( 'admin_notices', 'perflab_plugin_admin_notices' );
+
+/**
+ * Callback function that print plugin activation script.
+ *
+ * @since n.e.x.t
+ */
+function perflab_print_plugin_activation_script() {
+	$js = <<<JS
+( function( $ ) {
+	$( document ).ajaxComplete( function( event, xhr, settings ) {
+		// Check if this is the 'install-plugin' request.
+		if ( settings.data && typeof settings.data === 'string' && settings.data.includes( 'action=install-plugin' ) ) {
+			var target_element = $( event.target.activeElement );
+			if ( ! target_element ) {
+				return;
+			}
+			/*
+				* WordPress core uses a 1s timeout for updating the activation link,
+				* so we set a 1.5 timeout here to ensure our changes get updated after
+				* the core changes have taken place.
+				*/
+			setTimeout( function() {
+				var plugin_url = target_element.attr( 'href' );
+				if ( ! plugin_url ) {
+					return;
+				}
+				var nonce = target_element.attr( 'data-plugin-activation-nonce' );
+				var plugin_slug = target_element.attr( 'data-slug' );
+				var url = new URL( plugin_url );
+				url.searchParams.set( 'action', 'perflab_activate_plugin' );
+				url.searchParams.set( '_wpnonce', nonce );
+				url.searchParams.append( 'plugin_slug', plugin_slug );
+				target_element.attr( 'href', url.href );
+			}, 1500 );
+		}
+	} );
+} )( jQuery );
```

Comment:
Reformatting with Prettier:

```suggestion
(function ($) {
  $(document).ajaxComplete(function (event, xhr, settings) {
    // Check if this is the 'install-plugin' request.
    if (
      settings.data &&
      typeof settings.data === "string" &&
      settings.data.includes("action=install-plugin")
    ) {
      var target_element = $(event.target.activeElement);
      if (!target_element) {
        return;
      }
      /*
       * WordPress core uses a 1s timeout for updating the activation link,
       * so we set a 1.5 timeout here to ensure our changes get updated after
       * the core changes have taken place.
       */
      setTimeout(function () {
        var plugin_url = target_element.attr("href");
        if (!plugin_url) {
          return;
        }
        var nonce = target_element.attr("data-plugin-activation-nonce");
        var plugin_slug = target_element.attr("data-slug");
        var url = new URL(plugin_url);
        url.searchParams.set("action", "perflab_activate_plugin");
        url.searchParams.set("_wpnonce", nonce);
        url.searchParams.append("plugin_slug", plugin_slug);
        target_element.attr("href", url.href);
      }, 1500);
    }
  });
})(jQuery);
```

Reply:
I'm not sure we'll want to do that. Did you use regular Prettier? It doesn't follow the WordPress JS coding standards I believe (see e.g. all the missing spaces).

Reply:
Right. I used regular Prettier. I've updated the suggestion to just fix the comment indentation.